src/templates/customTemplate/customTemplate.tsx

```diff
@@ -190,62 +215,70 @@ export const CustomTemplate: FunctionComponent<TemplateProps<CustomTemplateCerti
       <FlipCard isFlipped={isFlipped}>
         <FlipCardFace isFront={true}>
           <Profile>
+            <LicenseName className="text-uppercase">
+              {document.name} (UAPL)
```

Comment:
Is hardcoding `(UAPL)` the best way? What if the licence name changes?

Reply:
oh no, we don't need it. i am able to add this (UAPL) in json file